install-apps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# This script assumes to be located in /IONOS as submodule within the Nextcloud server
+# repository.
+
+BDIR="$( dirname "${0}" )"
+
+. "${BDIR}"/installed-apps.inc.sh
+
+ooc() {
+	php occ \
+		"${@}"
+}
+
+fail() {
+	echo "${*}"
+	exit 1
+}
+
+main() {
+	if ! which jq >/dev/null 2>&1; then
+		fail "Error: jq is required"
+	fi
+
+	echo "Install and enable apps ..."
+
+	for app in ${ENABLED_APPS}; do
+		echo "Install app '${app}' ..."
+		ooc app:install "${app}"
```

Comment:
We should fail if app install is not possible.
A-lá
`		if [ "${?}" -ne 0 ]; then
			fail "Error: failed to install app '${app}'"
		fi
`